src/services/services.ts

```diff
@@ -970,6 +970,7 @@ module ts {
         kind: string;
         kindModifiers: string;
         matchKind: string;
+        isCaseSensitive: boolean;

```

Comment:
To tell the host if a match is case sensitive or not.  Case sensitive matches of the same kind are considered 'better'.  This is used in VS for sorting items appropriately.